serializer/serializer.h

```diff
@@ -73,6 +73,17 @@ int serializer_logger_release();
 status_t ta_get_info_serialize(char** obj, ta_config_t* const ta_config, iota_config_t* const tangle,
                                ta_cache_t* const cache);
 
+/**
+ * @brief Serialze single number with specific key
```

Comment:
Typo, `Serialize`
It looks like serializing an identity object, not a single number. :thinking:

Reply:
Yes, I forget to modify the comment.